rest/route/github.go

```diff
@@ -337,14 +337,29 @@ func (gh *githubHookApi) handleGitTag(ctx context.Context, event *github.PushEve
 		}))
 		return err
 	}
-
+	token, err := gh.settings.GetGithubOauthToken()
+	if err != nil {
+		return errors.New("error getting github token")
+	}
 	pusher := event.GetPusher().GetName()
 	tag := model.GitTag{
 		Tag:    strings.TrimPrefix(event.GetRef(), refTags),
 		Pusher: pusher,
 	}
 	ownerAndRepo := strings.Split(event.Repo.GetFullName(), "/")
-	hash := event.HeadCommit.GetID()
+	hash, err := thirdparty.GetTaggedCommitFromGithub(ctx, token, ownerAndRepo[0], ownerAndRepo[1], tag.Tag)
```

Comment:
Is there something we can do to make sure that ownerAndRepo[1] won't panic? Maybe check that the array length is at least 2?

Reply:
This actually does happen in validatePushTagEvent